package.json

```diff
@@ -1,16 +1,15 @@
 {
-  "name": "gatsby-starter-default",
-  "description": "Gatsby default starter",
+  "name": "@neontribe/www",
+  "description": "Neontribe website built with Gatsby",
   "version": "1.0.0",
-  "author": "Kyle Mathews <mathews.kyle@gmail.com>",
```

Comment:
lol - how did these get in in the first place?

Reply:
They come from the Gatsby generator

Reply:
Of course but was wondering how approved that they get in to overwrite the already initialised gatsby in PR #1